build.sbt

```diff
@@ -1,11 +1,13 @@
-import BuildHelper._
+enablePlugins(ZioSbtCiPlugin, ScalafixPlugin)
+
+crossScalaVersions := Seq.empty
 
 inThisBuild(
   List(
-    organization := "dev.zio",
-    homepage := Some(url("https://zio.dev/zio-config/")),
-    licenses := List("Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0")),
-    developers := List(
+    name                       := "ZIO Config",
+    crossScalaVersions -= scala211.value,
```

Comment:
I would rather just rely on ecosystem plugin to decide the common scala versions that an ecosystem library should support. Obviously, ecosystem libraries can choose to exclude from this common list. This implies there will not be specific `inThisBuild` settings for crossScalaVersions. The exclusion, should be within each module.
Ecosystem Plugin can also choose to pre-built excluded list.

Reply:
Currently, zio-sbt provides default scala versions, then we can add/remove and customize that for the whole modules or any submodule. 
I'm not sure I got what you said, here I excluded scala 211 from all modules, although we can do that one by one.